Sources/EpoxyCore/SwiftUI/LayoutUtilities/IdealHeightMeasurementContainer.swift

```diff
@@ -0,0 +1,138 @@
+// Created by Bryn Bodayle on 1/24/22.
+// Copyright © 2022 Airbnb Inc. All rights reserved.
+
+import SwiftUI
+
+// MARK: - IdealHeightMeasurementContainer
+
+/// A view that has an `intrinsicContentSize` of the `view`'s `systemLayoutSizeFitting(…)` and
+/// supports double layout pass sizing and content size category changes.
+/// This container view uses an injected proposed width to measure the view and return its ideal
+/// height through the `SizingContext` binding.
+public final class IdealHeightMeasurementContainer<SwiftUIView, UIViewType>: UIView
+  where
+  UIViewType: UIView
+{
+
+  // MARK: Lifecycle
+
+  public init(view: SwiftUIView, uiView: UIViewType, context: SizingContext) {
+    self.view = view
+    self.uiView = uiView
+    self.context = context
+    super.init(frame: .zero)
+
+    addSubview(uiView)
+    setUpConstraints()
+  }
+
+  @available(*, unavailable)
+  public required init?(coder _: NSCoder) {
+    fatalError("init(coder:) has not been implemented")
+  }
+
+  // MARK: Public
+
+  public var view: SwiftUIView
+
+  public var uiView: UIViewType {
+    didSet { updateView(from: oldValue) }
+  }
+
+  public override var intrinsicContentSize: CGSize {
+    if let size = latestMeasuredSize {
+      return size
+    }
+
+    return measureView()
+  }
+
+  public override func invalidateIntrinsicContentSize() {
+    latestMeasuredSize = nil
```

Comment:
Added this so we can remeasure by calling `invalidateIntrinsicContentSize`

Reply:
Sounds good to me!